.codeclimate.yml

```diff
@@ -2,6 +2,7 @@ version: "2"         # required to adjust maintainability checks
 exclude_patterns:
   - "!**/test/"
   - "!db/**/*"
+  - "!lib/modules/smart_scheduling_links/test/*"
```

Comment:
Shouldn't this be covered in the line 2 up from here?  Does that need to be `"!**/test/*"` or is something wrong with your setup?

Reply:
Removed